pantheon/src/main/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlocksSubCommand.java

```diff
@@ -135,4 +132,100 @@ public void run() {
       }
     }
   }
+
+  /**
+   * blocks dump sub-command
+   *
+   * <p>Dump a specific block from storage
+   */
+  @Command(
+      name = "dump",
+      description = "This command dump a specific block from storage",
+      mixinStandardHelpOptions = true)
+  static class DumpSubCommand implements Runnable {
+    @SuppressWarnings("unused")
+    @ParentCommand
+    private BlocksSubCommand parentCommand; // Picocli injects reference to parent command
+
+    @Option(
+        names = "--number",
+        required = true,
+        paramLabel = MANDATORY_LONG_FORMAT_HELP,
+        description = "Number of the block to dump",
+        arity = "1..1")
+    private final Long blockNumber = null;
+
+    @Option(
+        names = "--to",
+        paramLabel = MANDATORY_FILE_FORMAT_HELP,
+        description = "File to write the block instead of standard output",
+        arity = "1..1")
+    private File dumpExportFile = null;
```

Comment:
`dumpExportFile` -> `blocksExportFile`

Reply:
I just made the requested changes here https://github.com/PegaSysEng/pantheon/pull/1641